teams_accounts.go

```diff
@@ -48,6 +48,7 @@ type TeamsAccountSettings struct {
 
 type BrowserIsolation struct {
 	UrlBrowserIsolationEnabled bool `json:"url_browser_isolation_enabled"`
+	NonIdentityEnabled         bool `json:"non_identity_enabled"`
```

Comment:
i would recommend updating both of these fields to match the convention at https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans to avoid future you some issues (at least in terraform land)

Reply:
@jacobbednarz oh I remember running into this before..
isn't that a breaking change though?

Reply:
it is however:

- it mitigates a bunch of zero values elsewhere; and
- we're (technically) pre 1.0 of the library so this sort of thing is fine within semver

Reply:
should I fix the changelog to reflect this?

Reply:
already done :) if you refresh you should see it now.